kdc/kerberos5.c

```diff
@@ -929,18 +929,21 @@ struct kdc_patypes {
 #define PA_ANNOUNCE	1
 #define PA_REQ_FAST	2 /* only use inside fast */
 #define PA_SYNTHETIC_OK	4
+#define PA_NOT_IN_FAST	8 /* only use outside fast */
```

Comment:
Do we really need `PA_NOT_IN_FAST`?

I'm thinking I want `PA_REQ_FAST_PREF` so we can have a migration strategy for realms where FAST is not deployed.